services/login_service/app/config/settings.py

```diff
@@ -1,15 +1,28 @@
+import os
+
+
 class Config:
     """
     Config class to store configuration variables like token expiration and database information.
     These can be set as environment variables or hardcoded for local development.
     """
 
-    # Token expiration time (in seconds) - e.g., 24 hours
-    JWT_ACCESS_TOKEN_EXPIRES = 24 * 3600
-
     # DB variables
     PROJECT_ID = "annular-net-436607-t0"
     DATASET_ID = "Instaspy_DS"
     TABLE_ID = "users"
 
     DEBUG = True
+
+    # keycloak
+    KEYCLOAK_SERVER_URL = "https://35.190.204.129:8443/"
+    KEYCLOAK_REALM = "login_signup"
+    KEYCLOAK_CLIENT_ID = "login-signup-backend-vm"
+    KEYCLOAK_CLIENT_SECRET = "olhdkK25NInloZ4IE47D0z7mlabHqZ4A"
+    KEYCLOAK_ADMIN_USER = "admin"
+    KEYCLOAK_ADMIN_PASSWORD = "admin"
```

Comment:
This is bad. Remove this. Also change client secret, and keycloak admin password, because even if you remove it from here it will stay in git history

Reply:
what do you mean by remove this? remove the file/add it to git ignore?